README.md

```diff
@@ -1 +1,223 @@
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Use ES6 modules and webpack**
+- **Create it dynamically** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Clone this repo. 
+- Change whatever you like.
+
+
+### Prerequisites
+
+To run this project you need: To clone or fork and run it in a browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/otmaneechchafyky/To-Do-list.git
+```
+
+using Linux or ubuntu:
+
+```sh
+  cd my-folder
+  git clone https://github.com/otmaneechchafyky/To-Do-list.git
+```
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+### Install
+
+- n/a
+
+### Usage
+
+To run the project, open with **index.html** with any web browser.
+
+For example using Linux:
+
+```sh
+  cd To-Do-List
+  google-chrome index.html
+```
+
+### Run tests
+
+- n/a
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+
+
+👤 **Otmane Echchafyky**
+
+
+- GitHub: [@Otmaneechchafyky](https://github.com/otmaneechchafyky/)
+- LinkedIn: [@Otmaneechchafyky](https://www.linkedin.com/in/otmane-echchafyky-125801248/)
+- Twitter: [@EchchafykyO](https://twitter.com/EchchafykyO)
+
+
+
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+Future changes:
+
+- **Make the project responsive**
+- **Add some animation** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/otmaneechchafyky/To-Do-list/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+- Thanks to all Microverse community
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) 
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+-->
+
```

Comment:
- [ ] Kindly deleted comments like this, as it serves as guides and should not be committed to your code base 👍